src/screens/Settings/NotificationSettings/components/PreferenceControls.tsx

```diff
@@ -141,7 +141,9 @@ export function Inner({
       {'filter' in preference && (
         <>
           <Divider />
-          <Text style={[a.font_bold, a.text_md]}>From</Text>
+          <Text style={[a.font_bold, a.text_md]}>
+            <Trans>From</Trans>
+          </Text>
           <Toggle.Group
             type="radio"
             label={_('Filter who you receive notifications from')}
```

Comment:
```suggestion
            label={_(`Filter who you receive notifications from`)}
```
I noticed while I was reviewing the PR that this string was surrounded by single quotes `''` rather than backticks `` but I thought Lingui would still extract it.

However, it doesn't appear in the list of new strings added so I guess it does need to have backticks for it to be translatable:

![IMG_0762](https://github.com/user-attachments/assets/f19c1d0b-26c6-4139-b41d-a651ea792e1e)